src/components/Profile/StakingTokens.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useCallback, useMemo } from 'react'
+import { useHistory } from 'react-router-dom'
+import {
+  Box,
+  Distribution,
+  GU,
+  shortenAddress,
+  useTheme,
+  useViewport,
+} from '@1hive/1hive-ui'
+
+import BigNumber from '@lib/bigNumber'
+import { useWallet } from '@providers/Wallet'
+import { stakesPercentages } from '@utils/math-utils'
+import { getNetworkType } from '@/utils/web3-utils'
+import { useGardens } from '@/providers/Gardens'
+
+const DISTRIBUTION_ITEMS_MAX = 6
+
+type StakeItem = {
+  amount?: any
+  gardenId: string | null
+  proposalId: string | null
+  proposalName: string
+}
+
+type TransformedStakeType = {
+  item: StakeItem
+  percentage: any
+} | null
+
+type StakingTokensProps = {
+  myStakes: Array<StakeItem>
+}
+
+function displayedStakes(stakes: Array<StakeItem>, total: BigNumber) {
+  return stakesPercentages(
+    stakes.map(({ amount }) => amount),
+    {
+      total,
+      maxIncluded: DISTRIBUTION_ITEMS_MAX,
+    }
+  ).map((stake: any) => ({
```

Comment:
Looking at the deployment I think we are calculating the percentages wrong as we are considering all the stakes across every Garden. Instead, I think we should consider the stake percentages per Garden.

Reply:
> Looking at the deployment I think we are calculating the percentages wrong as we are considering all the stakes across every Garden. Instead, I think we should consider the stake percentages per Garden.

+1 on this comment just noticed the same

Reply:
Good catch. I will fix it.

Reply:
<img width="282" alt="Captura de ecrã 2022-03-26, às 16 12 42" src="https://user-images.githubusercontent.com/13184907/160248044-3f29076b-0e29-4235-b316-cc8e3ae727f1.png">

made the fix

Reply:
Looking great! I think we can display the Inactive proposals stake the same way: per Garden and using % instead of token amount. What do you think?

Reply:
Will do it.

Reply:
@0xGabi I have added the inactive and change the background to a gray one. Also change the title of the block to "Token distribution per Garden" since it included "Active" and "Inactive"

![Captura de ecrã 2022-03-28, às 11 58 32](https://user-images.githubusercontent.com/13184907/160384210-34522d88-2bae-41ce-b061-f74ccfcb8063.png)

Reply:
Well done, I like how to merge both view!